fdbserver/workloads/ConsistencyCheck.actor.cpp

```diff
@@ -1293,6 +1327,40 @@ struct ConsistencyCheckWorkload : TestWorkload {
 				}
 			}
 
+			if (SERVER_KNOBS->CONSISTENCY_CHECK_ROCKSDB_ENGINE || SERVER_KNOBS->CONSISTENCY_CHECK_SQLITE_ENGINE) {
+				std::unordered_map<UID, KeyValueStoreType> storageTypeMapping =
+				    wait(self->getStorageType(storageServerInterfaces));
+
+				bool anySpecifiedEngine = false;
+				for (int j = 0; j < storageServers.size(); j++) {
+					auto ssStorageType = storageTypeMapping.find(storageServers[j]);
+					if (ssStorageType != storageTypeMapping.end()) {
+						if (SERVER_KNOBS->CONSISTENCY_CHECK_ROCKSDB_ENGINE &&
```

Comment:
just a nit.  we can do this outside the for loop and store the storageEngine and check here in avoiding two if's.